yarpp-template-unwind.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/*
+YARPP Template: SiteOrigin Unwind
+*/ ?>
+<h2 class="related-posts heading-strike"><?php esc_html_e( 'You may also like', 'siteorigin_unwind' ); ?></h2>
```

Comment:
The translation domain should be consistent. So always `siteorigin-unwind`, not `siteorigin_unwind`.